src/requestProcessor.ts

```diff
@@ -0,0 +1,3 @@
+export class AWEFAW {
```

Comment:
Huh? :)

Reply:
Sometimes the wrong file gets added to a commit... You know how those things go...
Deleted...